### PR TITLE
Index page/ exit updates

### PR DESCRIPTION
**Pull request by luntropy**

### CODE REVIEW

File: exit.php [hunk 1]
```diff
@@ -16,6 +16,7 @@
 
 # TO DO
 unset($_SESSION["user_id"]);
+$_SESSION["logout_msg"] = "Successful log out.";
 header("Location: index.php");
 
 $view = new View("exit");
```
Comment (yalishanda42) [inline exit.php]: This view will not be seen by the user as he will be redirected to `index.php`

**Review by yalishanda42 — CHANGES_REQUESTED** [review]

**Comment by allexks** [comment]
[Export](https://trello.com/c/y312888o/4-export)

**Comment by yalishanda42** [comment]
[Log Out](https://trello.com/c/H1Bufxjv/12-log-out)

File: exit.php (hunk 1 above)
Comment (luntropy) [reply exit.php]: Fixed.

**Pull request merged**